common/flatpak-utils-http.c

```diff
@@ -499,6 +499,11 @@ _write_cb (void *content_data,
         }
     }
 
+  /* Check for cancellation */
+  if (data->cancellable != NULL &&
+      g_cancellable_is_cancelled (data->cancellable))
```

Comment:
this can handle NULL, it can be simplified

Reply:
Simplified